docs/docs/operator/automatic-release-upgrades.md

```diff
@@ -0,0 +1,55 @@
+# Automatically upgrading the Operator (Experimental)
+
+The Kubescape operator provides an experimental capability to automatically fetch the latest chart version and upgrade the installed release.
+
+## How to Use
+
+To enable this capability, simply simply set the `capabilities.autoUpgrading` to `enable` and configure how often you would like to check for updates by adjusting the cron schedule:
+
+```yaml
+capabilities:
+  autoUpgrading: enable
+
+# [Omitted for brevity...]
+
+# If necessary, adjust when you want the Upgrader to check for updates and apply updates
```

Comment:
This is the first time the term Upgrader is referenced. I suggest adding a sentence in the opening naming it.

Reply:
I added a definition in the opening paragraph, should be fixed.